st2actions/st2actions/bootstrap/runnersregistrar.py

```diff
@@ -181,6 +181,12 @@
                                 'doesn\'t finish in timeout seconds.'),
                 'type': 'integer',
                 'default': FABRIC_RUNNER_DEFAULT_ACTION_TIMEOUT
+            },
+            'port': {
```

Comment:
I just realized Fabric supports host:port notation. I'll retain the behavior in the new runner as well.